app/ModelSerializers/Summit/Registration/PromoCodes/PrePaidSummitRegistrationDiscountCodeSerializer.php

```diff
@@ -0,0 +1,21 @@
+<?php namespace ModelSerializers;
+/**
+ * Copyright 2023 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+/**
+ * Class PrePaidSummitRegistrationDiscountCodeSerializer
+ * @package ModelSerializers
+ */
+class PrePaidSummitRegistrationDiscountCodeSerializer extends SummitRegistrationDiscountCodeSerializer
```

Comment:
why are this empty , is there is no implementation should be using the base class on ( SummitRegistrationPromoCodeSerializer) on registry and avoid this no op class

Reply:
done